### ISSUE TITLE
Feature Request: Turn on Webcam

### ISSUE THREAD

**Issue by matt-benett** [fredemmott/StreamDeck-Discord#100]
**Is your feature request related to a problem? Please describe.**
When I am playing a game, I can now join a channel (thank you), but it would also be nice to then be able to turn the web cam on.

**Describe the solution you'd like**
Button to enable webcam on/off within the current voice session

**Comment by cmccandless** [comment]
Just took a quick look into this, and I'm not sure it's currently possible with the RPC API. There doesn't seem to be a `camera` command: https://discord.com/developers/docs/topics/rpc#commands-and-events-rpc-commands

**Comment by Edstub207** [comment]
Yup, sadly can't do this even with hotekeys, which can be used to share screens. To be honest, I think it sadly makes sense because it could be exploited from a security perspective.
